src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -588,7 +600,7 @@ const NetworkModificationNodeEditor = () => {
                                 className={classes.list}
                                 onChecked={setSelectedItems}
                                 values={modifications}
-                                itemRenderer={(props) => (
+                                itemRenderer={(props) => {
```

Comment:
Why this change `()` -> `{;}` ?

Reply:
my bad i was using a console.log to test an forgot to clean the code.